phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/ViewSettings.java

```diff
@@ -250,7 +251,17 @@ private void initAboutPane() {
         gridPane.add(new TextWithStyle(translate("Git Revision:"), CAPTION_TITLE_CSS_CLASS), 0, 2);
         Hyperlink gitRevisionLink = new Hyperlink(applicationGitRevision);
         gitRevisionLink.setOnAction(event ->
-                hostServicesSupplier.get().showDocument("https://github.com/PlayOnLinux/POL-POM-5/commit/" + applicationGitRevision)
+                {
+                    String url = "https://github.com/PlayOnLinux/POL-POM-5/commit/" + applicationGitRevision;
+                    // use xdg-open until HostServices works
+                    Runtime runtime = Runtime.getRuntime();
+                    try {
+                        runtime.exec("xdg-open " + url);
```

Comment:
I know what you mean to say, but I think that this is unneeded in the case of opening the browser, because java itself has a function for that (`HostServices.showDocument`). I would agree to the suggestion to use a dedicated interface for this if it's needed for other tasks that aren't supported directly by java.